reana_client/cli/workflow.py

```diff
@@ -171,11 +171,10 @@ def workflow_create(ctx, file, name, skip_validation, access_token):
                         fg='red'), err=True)
         sys.exit(1)
     try:
-        reana_spec = load_reana_spec(click.format_filename(file),
-                                     skip_validation)
-
+        reana_specification = load_reana_spec(click.format_filename(file),
```

Comment:
Click needs to be updated to `Click>=7,<8`.

Reply:
Thanks!